doc/nrf/app_dev/device_guides/nrf54l/features.rst

```diff
@@ -14,10 +14,29 @@ For additional information, see the following documentation:
 * Zephyr page on the :ref:`zephyr:nrf54l15dk_nrf54l15`
 * :ref:`installation` and :ref:`configuration_and_build` documentation to install the |NCS| and learn more about its development environment.
 
+RISC-V core
+***********
+
+VPR core, named *fast lightweight peripheral processor* (FLPR), is designed to support the RISC-V instruction set, featuring key enhancements that enable efficient handling of basic and complex operations, as well as streamlined instruction processing.
+It can be used as either a standalone processor or as a helper core.
+As a helper processor, VPR specializes in managing tasks that require real-time attention or low power consumption, effectively boosting the performance of the main processor.
+It is versatile, capable of operating independently or as an integrated peripheral, with accessible components for easy control and customization.
+
+Trusted Firmware-M (TF-M)
+*************************
+
+Trusted Firmware-M provides a configurable set of software components to create a Trusted Execution Environment.
+It has replaced Secure Partition Manager as the solution used by |NCS| applications and samples.
+This means that when you build your application with CMSE enabled, the :ref:`TF-M <ug_tfm>` is automatically included in the build.
```

Comment:
```suggestion
This means that when you build your application with CMSE enabled, :ref:`TF-M <ug_tfm>` is automatically included in the build.
```
I don't think the link here is necessary. Could be on the introducing line instead, or taken out completely since the link is presented later for more information.